src/chariott_grpc.rs

```diff
@@ -35,6 +36,11 @@ impl ChariottServer {
         Self { registry: Arc::new(RwLock::new(registry)), broker }
     }
 
+    pub fn registry_do<T>(&self, f: impl FnOnce(&mut Registry<IntentBroker>) -> T) -> T {
+        let mut registry = self.registry.write().unwrap();
```

Comment:
I assume we `blindly` unwrap because we must have had a panic by another thread to run into an Error here?

Reply:
Correct. Concerns?

Reply:
No - just confirming.